tests/common/mock_cgroup_environment.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file was renamed from mock_cgroup_environment.py. The _MOCKED* arrays stay the same in the new file and the mocking logic is now moved to mock_environment.py